packages/frontend/amp/components/OnwardContainer.tsx

```diff
@@ -164,6 +171,11 @@ export const OnwardContainer: React.SFC<{
                                                 <MoustacheSection name="isAudio">
                                                     <VolumeHigh />
                                                 </MoustacheSection>
+                                                {/* <MoustacheSection name="isComment">
+                                                    <Quote />
+                                                </MoustacheSection> */}
```

Comment:
We can probably kill this commented out code

Reply:
Of course! Keep forgetting about commented code. Need to start reviewing my own prs before I submit them.